src/baguetteBox.js

```diff
@@ -313,8 +327,19 @@
         if (options.buttons === 'auto' && ('ontouchstart' in window || imagesMap[currentGallery].length === 1)) {
```

Comment:
Yeah, that is what this test is used for.

/CC @feimosi: maybe make a new issue to track this.